api/models/Detainees.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  autoPK: false,
+  attributes: {
+    person_id: {
```

Comment:
I know it's a person_id in the ICD but perhaps it'd be easier to use the built in `id` we get for free with sails and make sure it's a string? seems semantically a bit weird to have a `person_id` on a detainee